doc/other_stuff.md

```diff
@@ -0,0 +1,162 @@
+# Other important stuff
+
+CartoDB.js has many great features for you to use in your applications. Let’s take a look at some of the most important ones:
+
+## Viz JSON support
+
+The Viz.JSON document tells CartoDB.js all the information about your map, including the style you want to use for your data and the filters you want to apply with SQL. The Viz JSON file is served with each map you create in your CartoDB account.
+
+Although the Viz JSON file stores all your map settings, all these settings can be easily customized with CartoDB.js. For example, if you want to do something completely different than what you initially designed it for. Loading the Viz JSON is as simple as:
+
+<div class="code-title">Viz JSON support</div>
+{% highlight javascript %}
+cartodb.createVis('map', 'http://examples.cartodb.com/api/v2/viz/ne_10m_populated_p_1/viz.json')
+{% endhighlight %}
+
+## How to set a different host than cartodb.com
+CartoDB.js sends all requests to the cartodb.com domain by default. If you are running your own
+instance of CartoDB you can change the URLs to specify a different host.
+
+A different host can be configured by using ``sql_api_template`` and ``maps_api_template`` in the ``options`` parameter
+for any ``cartodb`` function call.
+
+The format of these templates is as follows:
+
+{% highlight javascript %}
+sql_api_template: 'https://{user}.test.com'
+{% endhighlight %}
+
+CartoDB.js will replace ``{user}``.
+
+Notice that you don't need to set the path to the endpoint, CartoDB.js will set it automatically.
+
+## Bounds wrapper
+
+We have added an easy method to get the bounding box for any dataset or filtered query using the CartoDB.js library. The **getBounds** function can be useful for guiding users to the right location on a map or for loading only the right data at the right time based on user actions.
```

Comment:
This is also documented [here](https://github.com/CartoDB/cartodb.js/pull/725/files#diff-363d4745c1355c9340fcee8e9791e493R56). Perhaps we can remove this and complete the description (if needed on the other file)